botocore/docs/utils.py

```diff
@@ -124,3 +125,49 @@ def document_auto_populated_param(self, event_name, section, **kwargs):
             section = section.get_section('structure-value')
             if self.name in section.available_sections:
                 section.delete_section(self.name)
+
+
+class HideParamFromOperations(object):
+    """Hides a single parameter from multiple operations.
+
+    This method will remove a parameter from documentation and from
+    examples. This method is typically used for things that are
+    automatically populated because a user would be unable to provide
+    a value (e.g., a checksum of a serialized XML request body)."""
+    def __init__(self, service, parameter_name, operation_names):
+        """
+        :type parameter_name: str
+        :param parameter_name: Name of the parameter to modify.
+
+        :type operation_names: list
+        :param operation_names: Operation names to modify.
+        """
+        self._parameter_name = parameter_name
+        self._event_list = {}
+        # Build up a hash of relevant event names.
+        param_template = 'docs.request-params.%s.%s.complete-section'
+        example_template = 'docs.request-example.%s.%s.complete-section'
+        for name in operation_names:
+            self._event_list[param_template % (service, name)] = 'params'
+            self._event_list[example_template % (service, name)] = 'example'
+
+    def hide_param(self, event_name, section, **kwargs):
+        if event_name in self._event_list:
+            if self._event_list[event_name] == 'example':
+                section = section.get_section('structure-value')
```

Comment:
This changes the section that's being checked to see if the parameter is present. If it's an example, then we need to get the structure-value. Otherwise we need to use the section directly.